app/src/main/java/com/github/sdp/ratemyepfl/activity/course/CourseReviewActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.github.sdp.ratemyepfl.activity.course
+
+import android.os.Bundle
+import androidx.activity.viewModels
+import com.github.sdp.ratemyepfl.R
+import com.github.sdp.ratemyepfl.activity.ReviewActivity
+import com.github.sdp.ratemyepfl.viewmodel.ReviewViewModel
+import dagger.hilt.android.AndroidEntryPoint
+
+@AndroidEntryPoint
+class CourseReviewActivity: ReviewActivity() {
+
+    private val viewModel by viewModels<ReviewViewModel>()
```

Comment:
You should remove it as it is now used in `ReviewListFragment` and `ReviewInfoFragment`s (same for `RoomReviewActivity` and `RestaurantReviewActivity`)

Reply:
Are you talking about the view model ?

Reply:
Yes

Reply:
We use the viewModel to receive the id passed as an extra, so I need to put it here. I then share this viewModel with the different fragments.

Reply:
Oh ok my bad.